packages/parse-props/package.json

```diff
@@ -18,11 +18,11 @@
     "access": "public"
   },
   "dependencies": {
-    "@theme-ui/css": "0.11.3",
-    "@emotion/react": "^11.4.1"
```

Comment:
Will the `import { Interpolation } from '@emotion/react';` still work if we remove it completely (without adding it to peerDeps)?

Reply:
Actually, we should add an ESLint rule to catch this.

Reply:
Added `import/no-extraneous-dependencies`.